<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_Turkce">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Türkçe</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Sosyal_Bilimler">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sosyal Bilimler</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Matematik">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Matematik</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Fen_Bilimleri">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fen Bilimleri</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Turkce_D">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Sosyal_Bilimler_D">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Matematik_D">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Fen_Bilimleri_D">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Fen_Bilimleri_Y">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Sosyal_Bilimler_Y">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Matematik_Y">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Turkce_Y">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_D">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Y">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Y_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NET</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Sosyal_Bilimler_NET">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Fen_Bilimleri_NET">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Matematik_NET">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Turkce_NET">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Esittir_1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Esittir_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Esittir_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Esittir_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Calculation">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Hesapla</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ProgressBar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>391</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
